examples/basic/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+OREMDA_VAR_DIR=/run/oremda
+DATA_DIR=/home/alessandro/oremda_data
+DOCKER_SOCKET=/var/run/docker.sock
+
+docker run \
+  -v $OREMDA_VAR_DIR:$OREMDA_VAR_DIR \
```

Comment:
One thing to note: a docker container by default will only have 64 MB of shared memory available. If we want the plasma socket to have more shared memory, we can pass another argument to `docker run` like so:
```bash
--shm-size=750m
``` 

This increases the shared memory size of the docker container to 750 MB.

Reply:
Thank you, that's great to know for the future, when we have pipelines with real data!